components/CollectionCells/examples/CollectionCellsLayoutExample.m

```diff
@@ -166,6 +166,29 @@ - (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView
   return cell;
 }
 
+#pragma mark - <UICollectionViewDelegateFlowLayout>
+
+- (UIEdgeInsets)collectionView:(UICollectionView *)collectionView
```

Comment:
Yep, good callpout. I'm working on writing a new "Collections" example and an Earl Grey test to verify it.